relayer/chains/wasm/config.go

```diff
@@ -37,7 +37,7 @@ type Config struct {
 	SignModeStr            string        `json:"sign-mode" yaml:"sign-mode"`
 	Simulate               bool          `json:"simulate" yaml:"simulate"`
 	ExtraCodec             string        `json:"extra-codecs" yaml:"extra-codecs"`
-	BlockBatchSize         uint64        `json:"block-batch-size" yaml:"block-batch-size"`
+	BlockBatchSize         int           `json:"block-batch-size" yaml:"block-batch-size"`
```

Comment:
Why is the type changed here ? Wouldn't setting block size 1 in the config prepare the environment for disabling batch ?

Reply:
yeah make sense.

This decision I made was to make sure batch is explicitly disabled by user so that we invoke single wasm query.